brainstormforce/astra
# AST-3160 - Elementor Mini Cart Widget Conflict With Astra

### Description

- Astra Conflicts with Elementor Pro's Mini Cart Widget
- JIRA - https://brainstormforce.atlassian.net/browse/AST-3160

### Screenshots

[ele-cart-conflict](https://share.getcloudapp.com/bLuO980v) 
[Screencast at April 20th 2023 - 4.mp4](https://d.pr/v/Wd4VNp)

### Types of changes




Bug fixes

### How has this been tested?

- Activate Elementor Pro
- Add Mini Cart Widget
- Elementor > Integration > WooCommerce > Mini Cart > Enabled : The Qty Updater should not break.
- Elementor > Integration > WooCommerce > Mini Cart > Disabled : The Mini Cart Template should not break, UI should be well aligned.
- Check for RTL.

### Checklist:
